utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -630,6 +630,27 @@ int main(int argc, char **argv)
 
                     break;
                 }
+                case 11:
+                {
+                    testCaseName = "rain";
+
+                    Rpp32f rainPercentage = 7;
+                    Rpp32u rainHeight = 6;
+                    Rpp32u rainWidth = 1;
+                    Rpp32s slant = 0;
+                    Rpp32f alpha[batchSize];
+                    for(int i = 0; i < batchSize; i++)
```

Comment:
Same comment
Please add space after for
```suggestion
                    for (int i = 0; i < batchSize; i++)
```

Reply:
Done